src/js/components/List/List.js

```diff
@@ -562,6 +642,29 @@ const List = React.forwardRef(
                   }
                 }
 
+                // Pinned icon and settings
+                const Pin = theme.list.icons.pin;
+                const pinSize = theme.list.item.pinned.icon.size;
+                const pinPad = theme.list.item.pinned.icon.pad;
+
+                let displayPinned;
+                if (isPinned) {
+                  boxProps = {
+                    direction: 'row',
+                    align:
+                      (defaultItemProps && defaultItemProps.align) || 'center',
+                    gap: 'medium',
+                  };
+                  displayPinned = (
+                    <Box direction="row" align="center" justify="end">
+                      <Box pad={pinPad}>
```

Comment:
Do you still need two boxes here? I would think these could be combined.

Reply:
Ah sorry forgot to double check that. You're correct that Box is unnecessary.